examples/gno.land/r/gnoland/pages/admin.gno

```diff
@@ -0,0 +1,86 @@
+package gnopages
+
+import (
+	"std"
+	"strings"
+
+	"gno.land/p/demo/avl"
+)
+
+var (
+	adminAddr     std.Address
+	moderatorList avl.Tree
+	inPause       bool
+)
+
+func init() {
+	// adminAddr = std.GetOrigCaller() // FIXME: find a way to use this from the main's genesis.
+	adminAddr = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq"
+}
+
+func AdminSetAdminAddr(addr std.Address) {
+	assertIsAdmin()
+	adminAddr = addr
+}
+
+func AdminSetInPause(state bool) {
+	assertIsAdmin()
+	inPause = state
+}
+
+func AdminAddModerator(addr std.Address) {
+	assertIsAdmin()
+	moderatorList.Set(addr.String(), true)
+}
+
+func AdminRemoveModerator(addr std.Address) {
+	assertIsAdmin()
+	moderatorList.Set(addr.String(), false) // FIXME: delete instead?
```

Comment:
Are these FIXMEs something to be done before merging?

Reply:
```suggestion
	moderatorList.Set(addr.String(), false) // XXX: delete instead?
```

Yeah, not a fixme, more like an open question about best practices, thank you.